parachains/index.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Get Started
```

Comment:
Let's add some more context here for SEO purposes, like what are we getting started with?

Reply:
Added.